plugins/woocommerce/includes/gateways/paypal/class-wc-gateway-paypal.php

```diff
@@ -610,3 +610,42 @@ protected function should_use_orders_v2() {
 		);
 	}
 }
+
+/**
+ * Register PayPal scheduled actions on init.
+ * This ensures the action is available even when the gateway isn't instantiated.
+ */
+add_action( 'init', function() {
```

Comment:
I tried to register the action hook in the constructor of the gateway class ( `WC_Gateway_Paypal` ) and the webhook class. But it did not work. A job is created but it never gets triggered. If I manually run the job, it fails with the following error-

`action failed via Admin List Table: Scheduled action for woocommerce_paypal_capture_payment will not be executed as no callbacks are registered.`
So I had to register it in the `init` callback. Any other alternate solution is welcome.